packages/expo/bundledNativeModules.json

```diff
@@ -97,5 +97,7 @@
   "unimodules-app-loader": "~1.0.2",
   "expo-notifications": "~0.1.7",
   "expo-splash-screen": "~0.2.3",
-  "firebase": "7.9.0"
+  "firebase": "7.9.0",
+  "@react-native-community/picker": "1.5.1",
+  "@react-native-community/slider": "3.0.0-rc.2"
```

Comment:
Will this make its way from RC to a "final" version in the next couple of days? Otherwise unless there are serious issues with 2.x I think we should prefer the stable version.

Reply:
yeah i would go for 2.x if not. i have pinged the owners of this library on discord a couple times. my concern if we go with 2.x is that it doesn't have web support that @EvanBacon added, so i'm hoping that they will land 3.0 proper soon.